fulp_modules/features/sect/bloodthingie.dm

```diff
@@ -0,0 +1,62 @@
+/obj/item/reagent_containers/syringe/earth
+	name = "knife thing"
+	desc = "A syringe that can hold up to 150 units."
+	icon = 'icons/obj/medical/syringe.dmi'
+	base_icon_state = "syringe"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	icon_state = "syringe_0"
+	inhand_icon_state = "syringe_0"
+	worn_icon_state = "pen"
+	amount_per_transfer_from_this = 0
+	possible_transfer_amounts = 150
+	volume = 150
+	custom_materials = list(/datum/material/iron=10, /datum/material/glass=20)
+	sharpness = SHARP_EDGED
+
+
+
+/obj/item/reagent_containers/syringe/earth/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+
+/obj/item/reagent_containers/syringe/afterattack(atom/target, mob/user, proximity)
+	if(!reagents.total_volume)
+		to_chat(user, span_warning("[src] is empty! Right-click to draw."))
+	return
```

Comment:
CI might be getting mad because you're never calling parent here.
You can probably check if target is a mob and early return, assuming you don't want it to be injectible at ALL.